apps/addons/models.py

```diff
@@ -296,15 +295,14 @@ class Addon(amo.models.OnChangeMixin, amo.models.ModelBase):
     wants_contributions = models.BooleanField(default=False)
     paypal_id = models.CharField(max_length=255, blank=True)
     charity = models.ForeignKey('Charity', null=True)
-    # TODO(jbalogh): remove nullify_invalid once remora dies.
-    suggested_amount = DecimalCharField(
-        max_digits=8, decimal_places=2, nullify_invalid=True, blank=True,
+
+    suggested_amount = models.DecimalField(
+        max_digits=8, decimal_places=2, blank=True,
         null=True, help_text=_(u'Users have the option of contributing more '
                                'or less than this amount.'))
 
-    total_contributions = DecimalCharField(max_digits=8, decimal_places=2,
-                                           nullify_invalid=True, blank=True,
-                                           null=True)
+    total_contributions = models.DecimalField(max_digits=8, decimal_places=2,
+                                              blank=True, null=True)
```

Comment:
Not sure why we'd want `null=True`, but that's a question for another PR I guess ;)